### ISSUE TITLE
User Review for static version

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guide-graphql-client#22]
User Review:

Allow a person with a fresh mind to read the guide and execute the steps described in the guide, test whether each step is clear enough to understand, and no logic errors.

Review and walkthrough by a non-SME or a new user with a critical and fresh pair of eyes:

- [ ] Record the time it takes to go through the entire guide
- [ ] Test each step in the guide is clear and easy to understand
- [ ] Ensure no logic errors
- [ ] Report anything that does not work
- [ ] Report anything usability issues (confusing or hard to follow)
- [ ] Does the guide achieve basic understanding/working knowledge of the topic in the guide?

**Comment by rsherget** [comment]
Time taken: ~35 minutes
No Logic Errors found, everything ran fine
I feel I gained the basic understanding/working knowledge that the guide was meant to provide

One minor clarity issue I found:
In the Accessing the application section in the paragraph directly below `Try the query operations`, there is a sentence 
```
When the hostname is specified to system-java8, but you can try out the operations using the hostname system-java11 as well.
```
I had to read it a few times before realizing that it was going along with the sentence after that is separated by a line break. Not a huge issue but combining the sentences and removing the line break could slightly improve readability. Something like
```
When the hostname is specified to system-java8, but you can try out the operations using the hostname system-java11 as well, the response is similar to the following example:
```

Another very small thing. At the very end of the Accessing the application section, under the mutation operation subsection, its clear on how to add the note to a system service, but doesn't say where and how to access it. It could be useless information, but maybe add a line stating that you can view the note you added by querying the system you added it to. Maybe say there will be a new JSON element `"note"` with the note text you provided in the mutation operation.

Overall it was a very good guide and I enjoyed learning from it! Very well done!

**Comment by gkwan-ibm** [comment]
close
- the above suggestions were applied
